### PR TITLE
chore: aligned service versions with integration

### PR DESCRIPTION
**Pull request by mzedel**
to align with
https://github.com/mendersoftware/integration/blob/master/docker-compose.yml#L60
https://github.com/mendersoftware/integration/blob/master/docker-compose.yml#L270

### CODE REVIEW

**Comment by oldgiova** [comment]
LGTM, let's also ask @tranchitella because he's the release manager

**Review by oldgiova — APPROVED** [review]

**Comment by tranchitella** [comment]
@mzedel and @oldgiova, we are running 4.4.x in production now.
I prefer to release Mender 3.6 (LTS) with this version and leave the upgrade to 5.0.x in the next release cycle.
What do you think?

File: mender/values.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@ api_gateway:
   image:
     registry: docker.io
     repository: traefik
-    tag: v2.5
+    tag: v2.8
```
Comment (tranchitella) [inline mender/values.yaml:106]: This is okay.

File: mender/values.yaml [hunk 2]
```diff
@@ -60,7 +60,7 @@ mongodb:
     service:
       type: LoadBalancer
   image:
-    tag: "4.4.13-debian-10-r63"
+    tag: "5.0"
```
Comment (tranchitella) [inline mender/values.yaml:63]: This one I don't want to include in the release.

**Review by tranchitella — CHANGES_REQUESTED** [review]

**Review by tranchitella — APPROVED** [review]

**Comment by tranchitella** [comment]
Thanks @mzedel, I merged.

**Pull request merged**